koans/8-AboutApplyingWhatWeHaveLearnt.test.js

```diff
@@ -41,7 +41,13 @@ describe("About Applying What We Have Learnt", function() {
 
       /* solve using filter() & all() / any() */
 
-      expect(productsICanEat.length).toBe(FILL_ME_IN);
+      var aux = _(products).filter(function (x) { return x.containsNuts == false });
```

Comment:
Aux is not a really meaningful name, maybe something like `productsWithoutNuts` would explain the variable better. Or you can skip the intermediate value altogether and just chain two filters: `_(products).filter(...).filter(...)`. Or, yet another option is to combine both conditions into a single `filter`.